### PR TITLE
get better message for status when using apache

### PR DESCRIPTION
**Pull request by aleksandra-tarkowska**
https://trello.com/c/SOvbfoLq/57-docs-pr-omero-web-pr-re-web-start-stop-gunicorn
To test:

- `bin/omero config set omero.web.application_server wsgi`
- `bin/omero web start|status`

```
$ dist/bin/omero web start
Post-processed 'omeroweb.viewer.min.css' as 'omeroweb.viewer.min.css'
Post-processed 'omeroweb.viewer.min.js' as 'omeroweb.viewer.min.js'

0 static files copied to '/Users/ola/OMERO/ome.git/dist/lib/python/omeroweb/static', 616 unmodified, 2 post-processed.
Clearing expired sessions. This may take some time... [OK]
[ERROR] You are deploying OMERO.web using Apache and mod_wsgi. OMERO.web does not provide any management for daemon process which communicate with Apache child processes using UNIX sockets to handle a request. Please check Apache directly.
Generate apache config using 'omero web config apache' or 'omero web config apache24' and reload web server.

$ dist/bin/omero web status
OMERO.web status... [ERROR] You are deploying OMERO.web using Apache and mod_wsgi. OMERO.web does not provide any management for daemon process which communicate with Apache child processes using UNIX sockets to handle a request. Please check Apache directly.

```

### CODE REVIEW

**Comment by jburel** [comment]
Can be seen as a doc PR cc @hflynn

**Comment by aleksandra-tarkowska** [comment]
It is written in the doc http://www.openmicroscopy.org/site/support/omero5.2/sysadmins/unix/install-web/install-apache.html#running-omero-web-unix-linux

**Comment by jburel** [comment]
@aleksandra-tarkowska: it is after the PR freeze that's why I mentioned it.

**Comment by hflynn** [comment]
Actually, I think I'll open a matching docs PR, I can see that the not everyone might grasp the implications of what we have written already.

File: components/tools/OmeroPy/src/omero/plugins/web.py [hunk 1]
```diff
@@ -602,8 +602,12 @@ def status(self, args, settings):
             else:
                 self.ctx.err("[NOT STARTED]")
         elif deploy in (settings.WSGI,):
-            self.ctx.err("You are deploying OMERO.web using apache and"
-                         " mod_wsgi. Cannot check status.")
+            self.ctx.err("You are deploying OMERO.web using Apache and"
+                         " mod_wsgi. OMERO.web does not provide any "
+                         " management for ‘daemon’ process which "
```
Comment (rleigh-dundee) [inline components/tools/OmeroPy/src/omero/plugins/web.py]: for the

File: components/tools/OmeroPy/src/omero/plugins/web.py [hunk 2]
```diff
@@ -602,8 +602,12 @@ def status(self, args, settings):
...
+                         " communicate with Apache child processes"
```
Comment (rleigh-dundee) [inline components/tools/OmeroPy/src/omero/plugins/web.py]: communicates

**Comment by rleigh-dundee** [comment]
I can't currently run the code to test it, but the text reads fine.

**Comment by jburel** [comment]
```
test/unit/tablestest/test_servants.py::TestTables::testErrorInGet PASSED
     [exec] 
     [exec] =================================== FAILURES ===================================
     [exec] __________________________ TestWeb.testWebStart[wsgi] __________________________
     [exec] 
     [exec] self = <test_web.TestWeb object at 0x1070e350>, app_server = 'wsgi'
     [exec] monkeypatch = <_pytest.monkeypatch.monkeypatch instance at 0x10714cf8>
     [exec] capsys = <_pytest.capture.CaptureFixture instance at 0x10714dd0>
     [exec] 
     [exec]     @pytest.mark.parametrize('app_server', ['wsgi', 'wsgi-tcp', 'development'])
     [exec]     def testWebStart(self, app_server, monkeypatch, capsys):
     [exec]         self.mock_django_setting('APPLICATION_SERVER', app_server, monkeypatch)
     [exec]         self.set_python_dir(monkeypatch)
     [exec]         self.mock_subprocess_popen(monkeypatch)
     [exec]         self.mock_subprocess_call(monkeypatch)
     [exec]         self.set_django_pid(monkeypatch)
     [exec]         self.args += ["start"]
     [exec]         self.cli.invoke(self.args, strict=(app_server not in ('wsgi',)))
     [exec]         o, e = capsys.readouterr()
     [exec]         csout = "Clearing expired sessions. This may take some time... [OK]"
     [exec]         assert csout == o.split(os.linesep)[0]
     [exec]         if app_server in ('wsgi',):
     [exec]             stderr = (
     [exec]                 ("You are deploying OMERO.web using apache and mod_wsgi. "
     [exec]                  "Generate apache config using 'omero web config apache' "
     [exec]                  "or 'omero web config apache24' "
     [exec]                  "and reload web server."))
     [exec] >           assert stderr == e.split(os.linesep)[0]
     [exec] E           assert 'You are depl...d web server.' == '[ERROR] You a...le a request.'
     [exec] E             - You are deploying OMERO.web using apache and mod_wsgi. Generate apache config using 'omero web config apache' or 'omero web config apache24' and reload web server.
     [exec] E             + [ERROR] You are deploying OMERO.web using Apache and mod_wsgi. OMERO.web does not provide any management for the \u2018daemon\u2019 process which communicates with Apache child processes using UNIX sockets to handle a request.
     [exec] 
     [exec] test/unit/clitest/test_web.py:193: AssertionError
     [exec]  generated xml file: /home/travis/build/openmicroscopy/openmicroscopy/components/tools/OmeroPy/target/reports/junit-results.xml 
     [exec] ============== 1 failed, 1427 passed, 1 skipped in 135.72 seconds ==============
     [exec] INFO:om
```
travis failure

**Comment by aleksandra-tarkowska** [comment]
ohh test thx

**Comment by jburel** [comment]
Travis is green. Merging

**Pull request merged**